### PR TITLE
NET-219 Remove one of the testing frameworks

### PR DESCRIPTION
**Pull request by Vladacdc**

### CODE REVIEW

**Review by igolovchuk — CHANGES_REQUESTED** [review]
- do not name pull request as a branch name
- remove one of the testing frameworks means not just remove it and all related files, you should remove lib from the project, add lib that you will use and rewrite all tests to use that lib

**Comment by Vladacdc** [comment]
@igolovchuk 
> ...rewrite all tests to use that.

I have to rewrite them using DTO? 
If yes, then this task is blocked by  #6

**Comment by igolovchuk** [comment]
> @igolovchuk
> 
> > ...rewrite all tests to use that.
> 
> I have to rewrite them using DTO?
> If yes, then this task is blocked by #6

For now, you can commit as it was(without DTO), the task #6 owner will fix tests before megre in his pull request

File: TransIT.BLL.Tests/Services/CrudServiceTests.cs [hunk 1]
```diff
@@ -81,12 +80,20 @@ public async Task CreateAsync_GivenWrongEntity_ReturnsNull()
...
+            {
+                result = await _crudService.CreateAsync(entity);
+            }
+            catch (Exception)
+            {
+            }
+            finally
+            {
+                Assert.Null(result);
+                _repository.Verify(r => r.AddAsync(entity), Times.Once);
```
Comment (igolovchuk) [inline TransIT.BLL.Tests/Services/CrudServiceTests.cs]: - try to use framework for this eg.  
```
// act & assert
    Assert.Throws<Exception>(async () => await _crudService.CreateAsync(entity)); // or 
await Assert.ThrowsAsync<Exception>(async () => await _crudService.CreateAsync(entity));
   _repository.Verify(r => r.AddAsync(entity), Times.Once);
    VerifyLogger();
```

File: TransIT.BLL.Tests/Services/CrudServiceTests.cs [hunk 2]
```diff
@@ -159,11 +152,18 @@ public async Task DeleteAsync_GivenExistingEntityId_DeletesEntity(int id)
...
+            try
+            {
+                await _crudService.DeleteAsync(id);
 
-            await _crudService.DeleteAsync(id);
+            }
+            catch (Exception)
+            {
+
+            }
```
Comment (igolovchuk) [inline TransIT.BLL.Tests/Services/CrudServiceTests.cs]: - the same here, try to use framework for this, ex. look in the previous comment

File: TransIT.BLL.Tests/Services/ImplementedServices/CountryServiceTests.cs [hunk 3]
```diff
@@ -2,15 +2,14 @@
 using TransIT.DAL.Models.Entities;
 using TransIT.DAL.Repositories.InterfacesRepositories;
 
-namespace TransIT.BLL.NTests.Services.ImplementedServices
+namespace TransIT.BLL.Tests.Services.ImplementedServices
 {
-    public class CountryServiceTest : CrudServiceTest<Country>
+    class CountryServiceTests : CrudServiceTest<Country>
```
Comment (igolovchuk) [inline TransIT.BLL.Tests/Services/ImplementedServices/CountryServiceTests.cs]: add modifier

File: TransIT.BLL.Tests/Services/ImplementedServices/SupplierServiceTests.cs [hunk 4]
```diff
@@ -2,15 +2,14 @@
 using TransIT.DAL.Models.Entities;
 using TransIT.DAL.Repositories.InterfacesRepositories;
 
-namespace TransIT.BLL.NTests.Services.ImplementedServices
+namespace TransIT.BLL.Tests.Services.ImplementedServices
 {
-    public class SupplierServiceTest : CrudServiceTest<Supplier>
+    class SupplierServiceTests : CrudServiceTest<Supplier>
```
Comment (igolovchuk) [inline TransIT.BLL.Tests/Services/ImplementedServices/SupplierServiceTests.cs]: add modifier

File: TransIT.BLL.Tests/Services/ImplementedServices/CurrencyServiceTests.cs [hunk 5]
```diff
@@ -2,15 +2,14 @@
 using TransIT.DAL.Models.Entities;
 using TransIT.DAL.Repositories.InterfacesRepositories;
 
-namespace TransIT.BLL.NTests.Services.ImplementedServices
+namespace TransIT.BLL.Tests.Services.ImplementedServices
 {
-    public class CurrencyServiceTest : CrudServiceTest<Currency>
+    class CurrencyServiceTests : CrudServiceTest<Currency>
```
Comment (igolovchuk) [inline TransIT.BLL.Tests/Services/ImplementedServices/CurrencyServiceTests.cs]: add modifier

**Review by igolovchuk — CHANGES_REQUESTED** [review]
- see comments

**Review by igolovchuk — APPROVED** [review]

**Review by BohdanBuhrii — APPROVED** [review]

**Review by VitaliiMaksymiv — APPROVED** [review]

**Pull request merged**